docs/install/install-local-new.md

```diff
@@ -181,7 +181,8 @@ You now have the current ISLE project code checked into git as a foundation to m
 * Follow the in-line instructions to add your project's name to the appropriate areas.
     * Once finished, save and close the file.
 
-* _Using the same open terminal:_, navigate to "/pathto/yourprojectnamehere/scripts/proxy/ssl-certs/"
+* _Using the same open terminal:_
```

Comment:
Hi - I think you can change this back and just take out the comma. The first line `navigate to...` is meant as a summary with the `cd ... ` as how to do it.

Reply:
@bseeger no problem, I will change it back. Should I keep the comma right after the semicolon?

Reply:
good catch: please remove the comma after the semicolon.

Reply:
resolving this conversation as we need to push this into the docs for islandoracon.